src/styles/icon/icons.njk

```diff
@@ -0,0 +1,173 @@
+{% macro accountFilled(class) %}
+  <svg class="{{ class }}" xmlns="http://www.w3.org/2000/svg" height="2em" width="2em" viewBox="0 0 24 24">
```

Comment:
Should the `nhsapp-icon` class always be present on these svg elements, without the macro user having to specify it?

Reply:
The vue library does not hardcode this class os was just going off of that behaviour but happy to change it, no problem. Is this class always needed @davidhunter08

Reply:
It is applied to every icon so it makes sense not to make the user specify it.

Reply:
updated